bubuku/cli.py

```diff
@@ -79,6 +80,16 @@ def restart_broker(broker: str):
         RemoteCommandExecutorCheck.register_restart(zookeeper, broker_id)
 
 
+@cli.command('metrics', help='Get some important kafka metrics')
```

Comment:
do not really see a use case of that for now, maybe we can add it when we need it